### PR TITLE
Add support for "setHostname" to allow drop-in-replacement of Google Image Charts

### LINKED ISSUE #8: Google Image Chart deprecated 👎, Image-charts.com as a replacement 👍?
Hello @ryanrolds !

Awesome lib you did there! Since Google Image Chart could be shutdown at any time, we've built a **drop-in replacement** for it at image-charts.com. 

Would you be interested by a PR that adds support of Image-charts to Quiche? So users will be able to build their product on safe and futur-proof foundations?

### PR DESCRIPTION
**Pull request by FGRibreau**
Fixes #8 :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +105,7 @@
   * chart.setTitleSize(size [number]);
   * chart.setTitleRight();
   * chart.setTitleLeft();
+  * chart.setLegendOrder(order [string])
```
Comment (FGRibreau) [inline README.md]: this one was missing from the documentation :)

File: lib/api.js [hunk 2]
```diff
@@ -118,7 +119,9 @@ function processTitle(chart) {
 
   var title = chart.title;
   params.chtt = title.text;
-  params.chts = [title.color, title.size, title.align].join(',');
+  if(title.color || title.size || title.align){
```
Comment (FGRibreau) [inline lib/api.js]: fixes empty `chts=,,,` (see tests)

**Comment by FGRibreau** [comment]
Ping @ryanrolds :)

**Comment by ryanrolds** [comment]
I will review in the next 48 hours. In the middle of a v1.0.0 release of an application - so that has my full attention right now.

**Comment by ryanrolds** [comment]
Looks good.

**Comment by ryanrolds** [comment]
Thanks, released in v0.2.0.

**Comment by FGRibreau** [comment]
@ryanrolds awesome, thanks!

**Pull request merged**